frontend/src/metabase/querying/notebook/components/Notebook/Notebook.tsx

```diff
@@ -97,22 +98,26 @@ export const Notebook = ({
         {hasVisualizeButton && isRunnable && (
           <Flex
             py="sm"
-            pl="md"
+            pl="xl"
             style={{
               borderTop:
                 "var(--border-size) var(--border-style) var(--mb-color-border)",
             }}
           >
-            <Button
-              variant="filled"
-              radius="xl"
-              /* mah is a hack for 32px height button, we don't have it atm */
-              mah="xl"
-              miw={190}
-              onClick={visualize}
-            >
-              {t`Visualize`}
-            </Button>
+            {isResultDirty ? (
+              <Button
+                variant="filled"
+                radius="xl"
+                /* mah is a hack for 32px height button, we don't have it atm */
+                mah="xl"
+                miw={190}
+                onClick={visualize}
+              >
+                {t`Visualize`}
+              </Button>
+            ) : (
+              <ViewFooterControl question={question} isNotebook />
```

Comment:
:x: Because we're rendering `ViewFooterControl` in 2 different places, the component gets unmounted and mounted when switching from viz to notebook or from notebook to viz, which results in:
- non-continuous animation
- focus being lost (it goes back to the beginning of the page instead of staying within the control)


https://github.com/user-attachments/assets/bd1c9999-f109-4b2c-a5fc-c70cc140ec01

Reply:
it's not unmounted, it's hidden with css. I presume it's a result of some rendering optimization from the past. 

I also noticed weird animation and it must be fixed before merging 

https://github.com/metabase/metabase/blob/master/frontend/src/metabase/query_builder/components/view/View/NotebookContainer/NotebookContainer.tsx#L116-L120

Reply:
should be fixed now, I decided to render the same footer in both view and notebook editor